UI/src/stories/molecules/RipaStopDate.stories.js

```diff
@@ -0,0 +1,18 @@
+import RipaStopDate from '@/components/molecules/RipaStopDate'
+
+export default {
+  title: 'Molecules/RipaStopDate',
+  component: RipaStopDate,
+  parameters: {},
+}
+
+export const basic = () => ({
+  components: { RipaStopDate },
+  data() {
+    return {
+      stopDate
```

Comment:
I am getting an error that stopDate is not defined. Probably need to add:

`stopDate: {}`